lib/elixir/lib/enum.ex

```diff
@@ -2662,21 +2695,14 @@ defmodule Enum do
 
   ## slice
 
-  defp do_slice([], _start, _count) do
```

Comment:
it was only line L#2666 and L#2670 that were untouched by this commit. the rest of the lines were modified so i decided to make it compact as it is now